consensus/wal.go

```diff
@@ -83,7 +86,11 @@ func (wal *WAL) Save(wmsg WALMessage) {
 		}
 	}
 	// Write the wal message
-	var wmsgBytes = wire.JSONBytes(TimedWALMessage{time.Now(), wmsg})
+	innerMsgBytes := wire.JSONBytes(wmsg)
```

Comment:
I see similar code duplicated from `consensus/replay.go`, perhaps we can extract it to a common helper
```go
var crc32CastagnoliTable = crc32.MakeTable(crc32.Castagnoli)
func crc32Castagnoli(msg interface{}) (msgBytes []byte, crc uint32) {
   msgBytes = wire.JSONBytes(msg)
   crc = crc32.Checksum(msgBytes, crc32CastagnoliTable)
   return msgBytes, crc
}
```

Reply:
for 2 lines. don't think it's worth doing